python/test/unit/fem/test_element_integrals.py

```diff
@@ -440,7 +480,15 @@ def test_plus_minus_vector(cell_type, pm1, pm2, dtype):
 @pytest.mark.parametrize("pm1", ["+", "-"])
 @pytest.mark.parametrize("pm2", ["+", "-"])
 @parametrize_cell_types
-@pytest.mark.parametrize("dtype", [np.float32, np.float64, np.complex64, np.complex128])
+@pytest.mark.parametrize(
+    "dtype",
```

Comment:
Could we move the list of types to a single place at the top of the file?

Reply:
Will change.